client-app/shared/wishlists/components/add-to-wishlists-dialog.vue

```diff
@@ -73,44 +138,131 @@ const props = defineProps({
   },
 });
 
+const { t } = useI18n();
 const { openPopup, closePopup } = usePopup();
-const { loading: loadingLists, lists, fetchWishlists, addItemsToWishlists } = useWishlists({ autoRefetch: false });
+const {
+  loading: loadingLists,
+  lists,
+  fetchWishlists,
+  addItemsToWishlists,
+  createWishlistAndAddProduct,
+  removeItemsFromWishlists,
+} = useWishlists({ autoRefetch: false });
+const notifications = useNotifications();
 
 const loading = ref(false);
-const selectedListsIds = ref<string[]>([]);
+const selectedListsOtherIds = ref<string[]>([]);
+const listsRemove = ref<string[]>([]);
+const inputs = ref<WishlistInputType[]>([]);
+
+const productId = props.product.id;
+const config = inject(configInjectionKey);
+const listsLimit = config?.wishlists_limit || 10;
```

Comment:
It's preferred to move this to core/constants - magic numbers hidden in the code are not so clear.

Reply:
done